worker/src/RTC/DtlsTransport.cpp

```diff
@@ -149,6 +149,101 @@ namespace RTC
 			SSL_CTX_free(DtlsTransport::sslCtx);
 	}
 
+	DtlsTransport::Role DtlsTransport::RoleFromFbs(FBS::WebRtcTransport::DtlsRole role)
```

Comment:
Please @ibc, can you take a look at this file?

I've reviewed multiple times and all tests pass.

Reply:
I meant `file` not block :-)

Reply:
ok, next time click here to add a file comment:

![Screenshot 2023-09-11 at 12 19 37](https://github.com/versatica/mediasoup/assets/16191/b64fc7b1-8630-4d60-96f0-99d38f8a4c3b)

Reply:
File looks good.